autoload/editorconfig_core.vim

```diff
@@ -0,0 +1,138 @@
+" autoload/editorconfig_core.vim: top-level functions for
+" editorconfig-core-vimscript.
+" Copyright (c) 2018 Chris White.  All rights reserved.
+
+let s:saved_cpo = &cpo
+set cpo&vim
+
+" Variables {{{1
+
+" Note: we create this variable in every script that accesses it.  Normally, I
+" would put this in plugin/editorconfig.vim.  However, in some of my tests,
+" the command-line testing environment did not load plugin/* in the normal
+" way.  Therefore, I do the check everywhere so I don't have to special-case
+" the command line.
+
+if !exists('g:editorconfig_core_vimscript_debug')
+    let g:editorconfig_core_vimscript_debug = 0
+endif
+" }}}1
+
+" The version we are, i.e., the latest version we support
```

Comment:
This comment looks a bit weird. If it's about the latest version of the format we support, we should only have two numbers, because the third is for software update (bug fixes). I would just simply say "the version," because that's the least confusing and people should understand what it means.

Reply:
I am happy to change it.  Is there a wiki page or other document defining the project's versioning conventions?  I see that v0.12.2 is the PyPI version of editorconfig-core-py.  I also see that v0.12 is the latest version of editorconfig-core-test.  Does this mean that plugins should use the core-test version as their MAJOR.MINOR, but have their own PATCH?

The [core tests](https://github.com/editorconfig/editorconfig-core-test/blob/master/cli/CMakeLists.txt#L31) require the core to return a three-component version number.  Therefore, I have to use at least `0.12.0` to pass the tests.

What is the plan for the version numbering?  Thanks for helping me understand!

Reply:
@xuhdev Can we proceed with this while waiting for the general discussion in editorconfig issue 395?  vimcore currently passes all tests, including the version-related ones, and I believe this is the last open issue.  What do you think?

Reply:
Sorry, for some reason I didn't receive any notification about this. Perhaps it is the comment here that confused me ("the latest version we support" rather than "the version"). Which one do you intend it to be?

Reply:
@xuhdev I just updated my comment in editorconfig 395, and I updated the comment here in 53d117a7db6ba2c3da9efcf58b0734d011aadc37.  This is intended to be the specification version.  OK?

Reply:
Marked as resolved per your comment in https://github.com/editorconfig/editorconfig-vim/pull/119#pullrequestreview-238690349